src/cpx_model.jl

```diff
@@ -225,7 +225,7 @@ function set_warm_start!(model::Model, indx::IVec, val::FVec, effortlevel::Integ
 end
 
 function free_problem(model::Model)
-    tmp = Ptr{Cvoid}[model.lp]
+    tmp = model.lp
     stat = @cpx_ccall(freeprob, Cint, (Ptr{Cvoid}, Ptr{Cvoid}), model.env.ptr, tmp)
```

Comment:
Please in-line `tmp`, so that it is just
```julia
stat = @cpx_ccall(freeprob, Cint, (Ptr{Cvoid}, Ptr{Cvoid}), model.env.ptr, model.lp)
```

Also, can you run the tests `] test CPLEX` and confirm that they still all pass.